### PR TITLE
[Tensorflow][Test][experiment]Mnist and horovod test 

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*

*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py [hunk 1]
```diff
@@ -43,10 +43,8 @@ def test_tensorflow_train_mnist_gpu(tensorflow_training, ec2_connection, gpu_onl
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_MNIST_CMD)
 
 
-# TODO: Change this back TF_EC2_CPU_INSTANCE_TYPE. Currently this test times out on c4.8x, m4.16x and t2.2x,
-#       though passes on all three when run manually. For now we are pinning to c5.18 until we can resolve the issue.
 @pytest.mark.model("mnist")
-@pytest.mark.parametrize("ec2_instance_type", ["c5.18xlarge"], indirect=True)
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
```
Comment (YYStreet) [inline test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py:47]: May I know a context that why they are using "c5.18xlarge" as default originally?

File: test/dlc_tests/container_tests/bin/testBackendWithKeras [hunk 2]
```diff
@@ -19,7 +19,7 @@ TRAINING_LOG=${LOG_DIR}/${KERAS_BACKEND}_train_mnist.log
 sed -i 's/import keras/from tensorflow import keras/g; s/from keras/from tensorflow.keras/g' examples/mnist_cnn.py
 echo "Training mnist using $KERAS_BACKEND... This may take a few minutes. You can follow progress on the log file : $TRAINING_LOG"
 set +e
-python examples/mnist_cnn.py >$TRAINING_LOG 2>&1
+python examples/mnist_cnn.py 2>&1 | tee $TRAINING_LOG
```
Comment (YYStreet) [inline test/dlc_tests/container_tests/bin/testBackendWithKeras:22]: Does this code change work well?

File: test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py (hunk 1 above)
Comment (arjkesh) [reply test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py:47]: I believe the test was timing out on other instance types

**Comment by arjkesh** [comment]
nice fix, thanks!

File: test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py (hunk 1 above)
Comment (TusharKanekiDey) [reply test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py:47]: Yes, it was timing out for other instance types.

File: test/dlc_tests/container_tests/bin/testBackendWithKeras (hunk 2 above)
Comment (TusharKanekiDey) [reply test/dlc_tests/container_tests/bin/testBackendWithKeras:22]: Yes.

**Pull request merged**